book/src/products/mosaic/chains-supported-day-one.md

```diff
@@ -0,0 +1,21 @@
+# Chains and Assets Supported on Day One
+
+The chains/layers supported by Mosaic Phase 2 are as follows, enabling for active liquidity provisioning across these platforms:
+
+* [Ethereum Mainnet](https://ethereum.org/en/):
+* [Avalanche](https://www.avax.network/):
+* [Fantom](https://fantom.foundation/):
+* [Polygon](https://polygon.technology/):
+* [Moonriver](https://moonbeam.network/networks/moonriver/):
+* [Arbitrum](https://offchainlabs.com/): 
```

Comment:
These colons after list elements don't look like they should be there

Reply:
Noted.